instance/models/server.py

```diff
@@ -260,14 +260,20 @@ class OpenStackServer(Server):
     """
     A Server VM hosted on an OpenStack cloud
     """
+    openstack_region = models.CharField(max_length=16, blank=False)
```

Comment:
@smarnach We could mitigate that concern by showing the `server.openstack_region` in the GUI..  Though that was marked as "extra credit" for this issue, so is out of the required scope.

Once we get around to doing more GUI-related functionality updates, we can make these (and the other OpenStack) settings visible.